### ISSUE TITLE
OC 20477

### ISSUE THREAD

**Issue by QLLZ** [HopkinsIDD/cholera-database#597]

**Comment by eclee25** [comment]
Please also add a copy of the Promed report as a source document and the name of the report(s) in the Promed newsletter in the Source Notes. It looks like both the Moroto and Tororo reports were included in this OC.

**Comment by QLLZ** [comment]
yes, Moroto and Toroto reports were from two different websites, however the link to Toroto report has been deleted. so there is only one report in this OC.
